### PR TITLE
enable notebook on bigdl_0.3.0 and spark_2.2

### PR DESCRIPTION
**Pull request by chengxuHawkwood**
A few changes to notebook. @yiheng

### CODE REVIEW

**Comment by yangw1234** [comment]
Hi @lopelopelope, the changes in notebook is hard to review, so could you briefly summarize your changes here?

**Comment by chengxuHawkwood** [comment]
Hi. @yangw1234, I just change the utils.py and some steps in notebook to load images  to make the notebook work well without changing BigDL codes. Generally, I change some codes that are related to load MNIST. An error will occur in this notebook running on bigdl_0.2.1 later version.  However, zhichao is now updating the related codes, so maybe I should consider the changes again when zhichao finished his job.

**Comment by yangw1234** [comment]
LGTM

**Review by yangw1234 — APPROVED** [review]

**Comment by chengxuHawkwood** [comment]
I have changed the notebook according to zhichao's advice and the changes made in bigdl code. I  just added some to_ndarry methods@yangw1234 @yiheng @zhichao-li

**Comment by chengxuHawkwood** [comment]
@yiheng @yangw1234   PR ready to be reviewed.

File: README.md [hunk 1]
```diff
@@ -23,11 +23,11 @@ Step-by-step Deep Leaning Tutorials on Apache Spark using [BigDL](https://github
 + JDK 8
 + Apache Spark 2.1.0
 + Jupyter Notebook 4.1
-+ BigDL 0.2.0
++ BigDL 0.3.0
 + [Setup env on Mac OS](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupMac.md) / [Setup env on Linux](https://github.com/intel-analytics/BigDL-Tutorials/blob/master/SetupLinux.md)
 
 ### Start Jupyter Server
-* Download BigDL 0.2.0([linux64](https://repo1.maven.org/maven2/com/intel/analytics/bigdl/dist-spark-2.1.1-scala-2.11.8-linux64/0.2.0/dist-spark-2.1.1-scala-2.11.8-linux64-0.2.0-dist.zip), [mac](https://oss.sonatype.org/content/groups/public/com/intel/analytics/bigdl/dist-spark-2.1.1-scala-2.11.8-mac/0.2.0/dist-spark-2.1.1-scala-2.11.8-mac-0.2.0-dist.zip)) and unzip file.
+* Download BigDL 0.3.0 and unzip file.
```
Comment (yiheng) [inline README.md]: please add the link after we release the package

**Pull request merged**